modules/administration/pages/maintenance-window.adoc

```diff
@@ -1,36 +1,45 @@
 [[maintenance-window]]
-= Maintenance Window Checklist
+= Maintenance Window
 
-If you work on a {productname} Server with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+If you work with scheduled maintenance windows, you might find it difficult to remember all the things that you need to do before, during, and after that critical downtime of the {productname} Server.
+{productname} Server related systems such as Inter-Server Synchronization Slave Servers or {productname} Proxies are also affected and have to be considered.
 
-As with any {sle} system, you are encouraged to install package updates on the {susemgr} Server when they arrive.
+{suse} recommends you always keep your {productname} infrastructure updated.
+That includes servers, proxies, and build hosts.
+// It's similar to zypper at the package level:
+If you do not keep the {productname} Server updated, you might not be able to update some parts of your environment when you need to.
```

Comment:
Only the server? Maybe we should say better `{productname} infrastructure`?

Reply:
moio argued that it is especially important to keep the Server updated, the rest can follow.

Reply:
FMPOV: Server and Proxy, the rest can follow.

But if we mention build hosts, we should mention monitoring hosts as well (or whatever the name we have for the hosts where we install grafana and the prometheus server)